[2021-12-23 16:48:04,846] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T16:47:00+00:00 [queued]>
[2021-12-23 16:48:04,855] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.echo_meet_date 2021-12-23T16:47:00+00:00 [queued]>
[2021-12-23 16:48:04,859] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:48:04,860] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-23 16:48:04,860] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-23 16:48:04,875] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): echo_meet_date> on 2021-12-23T16:47:00+00:00
[2021-12-23 16:48:04,876] {standard_task_runner.py:52} INFO - Started process 351633 to run task
[2021-12-23 16:48:04,882] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'echo_meet_date', '2021-12-23T16:47:00+00:00', '--job-id', '1767', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpf064f4pe', '--error-file', '/tmp/tmp2z2y0yfs']
[2021-12-23 16:48:04,883] {standard_task_runner.py:77} INFO - Job 1767: Subtask echo_meet_date
[2021-12-23 16:48:04,926] {logging_mixin.py:104} INFO - Running <TaskInstance: test.echo_meet_date 2021-12-23T16:47:00+00:00 [running]> on host airflowvm
[2021-12-23 16:48:05,048] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=echo_meet_date
AIRFLOW_CTX_EXECUTION_DATE=2021-12-23T16:47:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-23T16:47:00+00:00
[2021-12-23 16:48:05,048] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-23 16:48:05,049] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo 211228']
[2021-12-23 16:48:05,052] {subprocess.py:75} INFO - Output:
[2021-12-23 16:48:05,053] {subprocess.py:79} INFO - 211228
[2021-12-23 16:48:05,054] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-23 16:48:05,072] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=test, task_id=echo_meet_date, execution_date=20211223T164700, start_date=20211223T164804, end_date=20211223T164805
[2021-12-23 16:48:05,090] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-23 16:48:05,096] {local_task_job.py:151} INFO - Task exited with return code 0
